src/mmw/js/src/app.js

```diff
@@ -64,6 +65,34 @@ var App = new Marionette.Application({
 
     hideLoginModal: function() {
         $('#login').modal('hide');
+    },
+
+    userLogout: function() {
+        var csrftoken = helpers.Cookie.get('csrftoken');
+
+        $.ajax({
+            url: '/user/ajaxlogout',
+            type: 'GET',
+            beforeSend: function(xhr) {
+                xhr.setRequestHeader('X-CSRFToken', csrftoken);
+            }
+        }).always(function() {
```

Comment:
If the request fails I think this will still set the ui to show the user as logged out. That could put things in an inconsistent state. I could imagine thinking I'm logged out and then refreshing the page and finding that I'm logged back in.